plugins/content/article/model.schema

```diff
@@ -52,6 +52,41 @@
       "title": "Is this available?",
       "help": "If set to 'false' this will not be displayed"
     },
+    "_onScreen": {
+      "type": "object",
+      "title": "",
+      "properties": {
+        "_isEnabled": {
+          "type": "bool",
+          "default": "false",
+          "isSetting": true,
+          "inputType": {"type": "Boolean", "options": [true, false]},
+          "validators": [],
+          "title": "Enable",
+          "help": "If set to 'true', a CSS-Class will be added once the Element is scrolled into view."
```

Comment:
I think this would be worded better as 'If enabled, the CSS class(es) specified in the Classes field will be added to the article element when it is within the browser's viewport'